nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-documentation/src/test/java/org/apache/nifi/documentation/DocGeneratorTest.java

```diff
@@ -72,7 +74,12 @@ public void testProcessorLoadsNarResources() throws IOException, ClassNotFoundEx
     }
 
     private NiFiProperties loadSpecifiedProperties(final String propertiesFile, final String key, final String value) {
-        String file = DocGeneratorTest.class.getResource(propertiesFile).getFile();
+        String file = propertiesFile;
```

Comment:
Was this included on accident? Does not seem relevant to your ticket.

Reply:
I certainly don't remember making this change. I suspect it changed on master since I forked, but I'll check.

Reply:
Sorry, it was relevant, to get it working in my local environment, where someone had put spaces in the path, %20 in resource paths needed converting back to ' '